{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/config.py

```diff
@@ -21,6 +23,14 @@ class DefaultProfile(object):
 
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
+    # For PGSQL applications, it might be good to uncomment this
```

Comment:
Go ahead and uncomment since most of our apps are PostgreSQL?  Leave a comment indicating it can be removed if not using PostgreSQL.